src/examples/proteopedia-wrapper/index.ts

```diff
@@ -386,7 +386,10 @@ class MolStarProteopediaWrapper {
             if (!options?.hideLabels) {
                 // Labels
                 const waters = MS.struct.generator.atomGroups({
-                    'entity-test': MS.core.rel.eq([MS.struct.atomProperty.macromolecular.entityType(), 'water']),
+                    'entity-test': MS.core.str.match([
+                        MS.re('water', 'i'),
```

Comment:
``^water$`` for this to be equivalent?

Or should we add ``lower-case`` function instead?